src/s_exp/mina/response.clj

```diff
@@ -7,6 +7,12 @@
            (io.helidon.webserver.http ServerResponse)
            (java.io FileInputStream InputStream OutputStream)))
 
+(def ^:no-doc ring-core-loaded
```

Comment:
small nit: rename to `ring-core-loaded?`

Reply:
done, i tend to have that for pred functions.